### PR TITLE
Login feature test

### PR DESCRIPTION
**Pull request by Prozak8**
### User Story
```
As a user,
I would like to navigate to login,
In order to login I need to enter my credentials.
```
------------
### Overview
- Installed Launchy for better understanding of feature test. Example; we had a problem with the 'given user exists' background and used launchy to find that we were applying no name to the login process.
- Added login feature, works fine.
------------
### What we've learnt
- Neroku has to reinstall if you add the launchy gem. Takes forever.
- Launchy is quite good. Have to use "Show me the page" within the feature steps in order to use gem to full extent. Also remember to take away before committing otherwise it messes up cucumber. 
- Steps can be reused if worded exactly the same.

### CODE REVIEW

File: features/login_to_account.feature [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Feature: Login
+
```
Comment (magnus-thor) [inline features/login_to_account.feature]: remove empty line

File: features/login_to_account.feature [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    As a user,
+    I would like to navigate to login,
+    In order to login I need to enter my credentials.
+
+Background:
```
Comment (magnus-thor) [inline features/login_to_account.feature]: indent the file properly

File: features/login_to_account.feature [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+| name      | email            | password |
+| Mike Hunt | mike@hotmail.com | password |
+
+Scenario: 'Login to my account'
+    Given I am on the landing page
+    When I click on "Login"
+    And I fill in "Email" with "mike@hotmail.com"
+    And I fill in "Password" with "password"
+    And I press 'Log in'
+    # Then show me the page
```
Comment (magnus-thor) [inline features/login_to_account.feature]: remove commented out code

File: features/step_definitions/all_step_definitions.rb [hunk 4]
```diff
@@ -1,16 +1,29 @@
 Given("I am on the landing page") do
   visit root_path
 end
+
+Given("the following user exist") do |table|
+  table.hashes.each do |person|
+  User.create!(person)
```
Comment (magnus-thor) [inline features/step_definitions/all_step_definitions.rb]: Indentation

File: features/step_definitions/all_step_definitions.rb [hunk 5]
```diff
@@ -1,16 +1,29 @@
...
+Then ("show me the page") do
+  save_and_open_page
+end
   
 When("I press {string}") do |button|
   click_on button
 end
-  
+
+When("I click on {string}") do |link|
```
Comment (magnus-thor) [inline features/step_definitions/all_step_definitions.rb]: Reuse the step above rather than creating a new identical step

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: features/login_to_account.feature [hunk 6]
```diff
@@ -0,0 +1,27 @@
+Feature: Login
+
+    As a user,
+    I would like to navigate to login,
+    In order to login I need to enter my credentials.
```
Comment (tochman) [inline features/login_to_account.feature]: Please revisit this user story. Together with a coach perhaps?

File: features/login_to_account.feature [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+Background: 
+Given the following user exist
+| name      | email            | password |
+| Mike Hunt | mike@hotmail.com | password |
```
Comment (tochman) [inline features/login_to_account.feature]: Code formatting is off here. Please install a Gherkin extension to yours VSCode and format code using command + F

File: features/step_definitions/all_step_definitions.rb (hunk 4 above)
Comment (tochman) [reply features/step_definitions/all_step_definitions.rb]: Please do not click "Resolve" on a comment without actually addressing the issue.

**Review by tochman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**